### DISCUSSION TITLE
change width of EditProfile page

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#9964] [Help] [answered]
I created a page which extends `Filament\Pages\Auth\EditProfile`

is there a way to change the width of the form? i would like to make it wider as there are a lot of fields.

**Comment by leandrocfe** [comment, marked as answer]
Add in the page created
```php
protected ?string $maxWidth = '2xl'; //lg, 2xl, 3xl... 
```

**Reply by jdion84** [reply]
thanks again leandro
